hoot-core/src/main/cpp/hoot/core/algorithms/linearreference/WaySublineMatchString.h

```diff
@@ -121,18 +121,7 @@ class WaySublineMatchString
    */
   bool touches(const WaySublineMatchString& other) const;
 
-  QString toString() const
-  {
-    QString result;
-    result += QString("matches:\n");
-    QStringList l;
-    for (size_t i = 0; i < _matches.size(); i++)
-    {
-      l.append(_matches[i].toString());
-    }
-    result += l.join("\n");
-    return result;
-  }
+  QString toString() const;
```

Comment:
Yup I saw it going from header to source and gave the ol' thumbs up.  We should probably draw straws to see who has to go through the whole code base and do that everywhere that makes sense.  But that is just me and the way I like to see header/source files.